.test_durations

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
Can we add this file to`.gitignore`?

Reply:
According to the documentation it looks like "pytest-spilt" uses this file to split up the test it the most efficient way.

https://jerry-git.github.io/pytest-split/

Reply:
Got it. 

1. Can we also use `--durations-path` to store in the `.github/` folder.? primarily to avoid cluttering the root folder. 

Looking at the docs, this doesn't indicate `pytest-split` as a replacement for running tests locally with `pytest-xdist`, or am I missing something here? It doesn't have a friendly CLI to split and run all the tests parallelly on the same machine. It seems pytest-split is more suited to CIs with multiple workers. That's why I was suggesting to keep the `.test_durations` file to kept in `.github` folder.